### ISSUE TITLE
Missing Header

### ISSUE THREAD

**Issue by mananbansal2002** [kognise/water.css#303]
I think there should be a proper header section in the website. The website name and its information are not very UX friendly and needs to be highlighted using CSS.

**Comment by erickrod54** [comment]
Hello @mananbansal2002, is still open this issue ?, i would like to make a nice UX friendly proper header section in the website, can you asign it to me ?

**Comment by Kamveno** [comment]
hi @mananbansal2002   i would like to contribute to the issue

**Comment by pranathnaik** [comment]
hi @mananbansal2002 would like to contribute to the issue

**Comment by developerpra** [comment]
Hello there, Assign this issue to me

**Comment by deepbeatz** [comment]
@mananbansal2002 please assign me this issue. I want to start working on it immediately.

**Comment by ritwick096** [comment]
Hello @mananbansal2002 please assign this issue to me. I shall start working on it immediately.

**Comment by TannisthaBiswas** [comment]
Since this issue is open still, I'd love to work on it. I'd work on it immediately making it more user friendly with an informative yet engaging.

**Comment by muhammadrehab1245** [comment]
@kognise  please assigned this to me i will do it

**Comment by RishikaKant** [comment]
Hello, please assign this to me

**Comment by 108aryanmaurya** [comment]
hey is this issue still open? if yes, then i wanna take this

**Comment by sudoarchie** [comment]
hey can you please assign this issue

**Comment by JuiPuranik** [comment]
hey can you please assign this issue

**Comment by JuiPuranik** [comment]
Im intrested in working on this issue. could you please assign this issue to me?

**Comment by riturajreal** [comment]
> I think there should be a proper header section in the website. The website name and its information are not very UX friendly and needs to be highlighted using CSS.

Yes I am also thinking that without a header, users may struggle to navigate the website.
